.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   #  Toolchains  #
   ################
   tc-gcc:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Could we pass the CI check? It looks like some CI jobs were canceled with this pull request.

Reply:
Thanks for the comment. In the new environment Verilator is now compiled using GCC.